src/doc/docUtil.js

```diff
@@ -360,11 +360,11 @@ export function getContext(pageEl, annotationLayerClass) {
     annotationLayerEl = scaleCanvas(pageEl, annotationLayerEl);
 
     const textLayerEl = pageEl.querySelector('.textLayer');
+    const canvasWrapperEl = pageEl.querySelector('.canvasWrapper');
     if (textLayerEl) {
         pageEl.insertBefore(annotationLayerEl, textLayerEl);
-    } else {
+    } else if (canvasWrapperEl) {
```

Comment:
Added a safety check for when a later page has not been rendered yet

Reply:
This safety check prevents errors that caused only some highlights to render on load and the others to render on click